From 41ac5624533d126b791feae79ed3df6f772057a4 Mon Sep 17 00:00:00 2001
From: Sam Miller <samjmill@us.ibm.com>
Date: Fri, 5 Jun 2009 15:31:54 -0500
Subject: [PATCH 03/53] tools: fix parallel build warning

Makefiles in src/apps and src/func_tests use 'make' instead
of $(MAKE) which causes failures in parallel builds.

Signed-off-by: Sam Miller <samjmill@us.ibm.com>
Signed-off-by: Vlad Yasevich <vladislav.yasevich@hp.com>
---
 src/apps/Makefile.am       | 4 ++--
 src/func_tests/Makefile.am | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/apps/Makefile.am b/src/apps/Makefile.am
index 0e6f9ee..7a0bbe5 100644
--- a/src/apps/Makefile.am
+++ b/src/apps/Makefile.am
@@ -19,10 +19,10 @@ noinst_PROGRAMS = bindx_test nagle_snd nagle_rcv myftp sctp_xconnect \
 	peel_server peel_client sctp_test sctp_status
 
 $(top_builddir)/src/lib/libsctp.la:
-	make -C $(top_builddir)/src/lib libsctp.la
+	$(MAKE) -C $(top_builddir)/src/lib libsctp.la
 
 $(top_builddir)/src/testlib/libsctputil.la:
-	make -C $(top_builddir)/src/testlib libsctputil.la
+	$(MAKE) -C $(top_builddir)/src/testlib libsctputil.la
 
 # Specifying the sources
 bindx_test_SOURCES = bindx_test.c
diff --git a/src/func_tests/Makefile.am b/src/func_tests/Makefile.am
index 44f0e94..e420bcb 100644
--- a/src/func_tests/Makefile.am
+++ b/src/func_tests/Makefile.am
@@ -62,10 +62,10 @@ PASSING_V6_KERN_TESTS = \
 noinst_PROGRAMS = ${PASSING_KERN_TESTS} ${PASSING_V6_KERN_TESTS}
 
 $(top_builddir)/src/lib/libsctp.la:
-	make -C $(top_builddir)/src/lib libsctp.la
+	$(MAKE) -C $(top_builddir)/src/lib libsctp.la
 
 $(top_builddir)/src/testlib/libsctputil.la:
-	make -C $(top_builddir)/src/testlib libsctputil.la
+	$(MAKE) -C $(top_builddir)/src/testlib libsctputil.la
 
 # These are tests for live kernels which pass.
 v4test: ${PASSING_KERN_TESTS}
-- 
1.7.11.7

